cpp/include/raft/random/rng.cuh

```diff
@@ -730,10 +749,10 @@ void laplace(const raft::handle_t& handle,
  *   the CUDA stream on which to run.
  * @param[inout] rng_state Pseudorandom number generator state.
  * @param[in] in Input vector to be sampled.
- * @param[in] wts Optional weights vector.
+ * @param[in] weights_opt std::optional weights vector.
  *        If not provided, uniform sampling will be used.
  * @param[out] out Vector of samples from the input vector.
- * @param[out] outIdx If provided, vector of the indices
+ * @param[out] outIdx_opt std::optional vector of the indices
```

Comment:
Sorry to be nitpicky but it just occurred to me that we also erase the template param type here. Can we specify that in the docs too? (e.g. std::optional<raft::device_vector_view<const WeightsT, IdxT>>).